WeakAuras/RegionTypes/DynamicGroup.lua

```diff
@@ -1084,6 +1084,9 @@ local function modify(parent, region, data)
       controlPoint:SetShown(show and frame ~= WeakAuras.HiddenFrames)
       controlPoint:SetWidth(regionData.dimensions.width)
       controlPoint:SetHeight(regionData.dimensions.height)
+      if data.anchorFrameParent then
+        controlPoint:SetParent(frame)
+      end
```

Comment:
You need to SetParent in the else case too.

Otherwise if a dynamic group that was previously set anchorFrameParent to true, will not work correctly once the options is disabled.